webapp-django/crashstats/crashstats/utils.py

```diff
@@ -286,13 +286,20 @@ def build_default_context(product=None, versions=None):
 
     # Build product information
     api = models.Products()
-    context['products'] = {
-        hit['product_name']: hit for hit in api.get()['hits']
-    }
+    # NOTE(willkg): using an OrderedDict here since Products returns products
+    # sorted by sort order and we don't want to lose that ordering
+    all_products = OrderedDict()
+    for item in api.get()['hits']:
+        if item['sort'] == -1:
```

Comment:
I thought about adding an `is_active` filter to the model, but in the (hopefully near) future, we want to move those sqlalchemy models to Django models where they will be a lot easier to do things with. Seemed like it didn't make sense to figure out how to do filters now and implement it that way.